Main_Project/config/application.props

```diff
@@ -0,0 +1,8 @@
+GET_SESSION_KEY_URL =https://italent2.demo.lithium.com/restapi/vc/authentication/sessions/login?user.login=saikumarn&user.password=Q!w2e3r4&restapi.response_format=json
+HTACCESS_HEAEDER_VALUE = Basic ZGVtbzAxOlN1UHIxKm1aTGk=
+BOARD_ID=java3-test
+CREATE_BOARD_URL=https://italent2.demo.lithium.com/restapi/vc/categories/id/Learn_Java_Lithium/boards/add?board.id=%BOARD_ID%&board.title=java1_test&restapi.session_key=%SESSION_KEY%&restapi.response_format=json
```

Comment:
"https://italent2.demo.lithium.com/restapi/vc/" this is common for GET_SESSION_KEY_URL, CREATE_BOARD_URL, MSG_URL, CHECK_BOARD, CREATE_MSG_REPLY, GETTING_USERS. so maintain one key this . remaining all fields add in code.